### DISCUSSION TITLE
Discrepancy in Address Data Between Komoot App and Self-Hosted Photon Instance

### DISCUSSION THREAD

**Discussion by philipredstone** [komoot/photon#818] [Q&A] [answered]
Hey,

I've been using the self-hosted version of Photon and noticed some discrepancies between the address data available in the Komoot app and the data provided through Photon. Specifically, I observed that certain addresses are searchable in the Komoot app but do not appear in Photon, even after a fresh import.

For example, the address "Kunibertstraße 15, 54497 Morbach" (in Germany) is searchable in the Komoot app but is not found in my self-hosted Photon instance. This suggests there might be additional address data or updates in the Komoot app that are not reflected in the Photon data.

I also tried to use the base planet.osm.pbf from OSM directly and imported it into Photon and even went as far as merging the daylight ml-buildings with the OSM data, but that didn't work either.

Can anyone provide some insights on the following please:

 - Are there additional data sources or updates used by the Komoot app that are not included in the Photon dataset?
 - How can I ensure that my self-hosted Photon instance has the same address data accuracy and completeness as the Komoot app?

Any guidance or technical details you can share would be greatly appreciated.

Thank you for your assistance. :)

**Comment by lonvia** [comment, marked as answer]
The Komoot app does not use Photon for searching, so differences in results are to be expected.

The example address cannot be found because OpenStreetMap does not have housenumbers [in the area](https://www.openstreetmap.org/#map=18/49.81087/7.03282).

**Reply by philipredstone** [reply]
hey @lonvia 

Thanks for clarifying that Komoot does not use Photon for search.

Could you please let me know which data sources or methods Komoot uses for address search functionality?

Thanks you! :)

**Reply by lonvia** [reply]
I wouldn't know.

**Reply by philipredstone** [reply]
okay, thanks anyway! :)
